lib/Controller/SAMLController.php

```diff
@@ -85,6 +85,8 @@ public function __construct($appName,
 	 */
 	private function autoprovisionIfPossible(array $auth) {
 		$uidMapping = $this->config->getAppValue('user_saml', 'general-uid_mapping');
+		$displayName = $auth['urn:oid:2.5.4.42'][0] . ' ' . $auth['urn:oid:1.2.40.0.10.2.1.1.261.20'][0];
```

Comment:
good point.
Two values like displayname.firstpart, displayname.secondpart to leave the admin a choice if first,last or last,first?